### PR TITLE
Rotate Medallions around in file select when all 6 have been obtained

### PR DESCRIPTION
**Pull request by GSKirox**
Just a small and funny Easter egg when you get all 6 meds.

https://github.com/OoTRandomizer/OoT-Randomizer/assets/65768236/37cf9011-0ea0-4a2a-8d07-f4201ba81839

### CODE REVIEW

File: ASM/c/file_icons.c [hunk 1]
```diff
@@ -348,6 +386,12 @@ static void populate_fixed(const z64_file_t* file, fixed_tile_info_t* info) {
 
     // medallions
     info->medallion_bits = (uint8_t)(file->quest_items & 0x0000003F);
+    hasAllSixMeds = (info->medallion_bits & (1 << QUEST_MEDALLION_FOREST) &&
+                     info->medallion_bits & (1 << QUEST_MEDALLION_FIRE) &&
+                     info->medallion_bits & (1 << QUEST_MEDALLION_WATER) &&
+                     info->medallion_bits & (1 << QUEST_MEDALLION_SPIRIT) &&
+                     info->medallion_bits & (1 << QUEST_MEDALLION_SHADOW) &&
+                     info->medallion_bits & (1 << QUEST_MEDALLION_LIGHT));
```
Comment (flagrama) [inline ASM/c/file_icons.c]: I'd take a look at how they did this for the ganon boss key condition.

Comment (GSKirox) [reply ASM/c/file_icons.c]: That is a lot more elegant than what i did, indeed. Thanks !

**Comment by ETR-BTF** [comment]
I don't like how this looks. Maybe if the cycling speed were slowed down to 60-120 frames per medallion change? But even then I think it looks messy and unclear.

**Comment by cjohnson57** [comment]
I personally like this, but it's the kind of thing I could see causing some undue strife lol. I'd post it in one of the dev channels and see what the reactions are. (Unless you already did that and I missed it)

**Comment by fenhl** [comment]
I could see it being very distracting and it would most likely have to be a cosmetic setting that people can turn off.

**Comment by ktspeedruns** [comment]
100% hate it. Looks like an epilepsy trigger.

**Comment by cjohnson57** [comment]
I'm not sure about that, I'm no expert but I don't think moving, static colors that take up that small of a portion of the screen is an epilepsy trigger

**Comment by fenhl** [comment]
This was discussed and voted on internally and we're going to reject it. My personal reasoning: This is too visually distracting to have it unconditionally enabled, and too minor to warrant adding a cosmetic setting for it.

**Pull request closed without merging**